src/containers/Tag/TagList/TagList.tsx

```diff
@@ -40,6 +45,14 @@ export const TagList: React.SFC<TagListProps> = (props) => {
     },
   });
 
+  useEffect(() => {
+    if (message.data && message.data.message) alert(message.data.message);
+    client.writeQuery({
```

Comment:
Looks like this is the main code to write to local state and we are using the same in multiple places. It might make sense to make a common method that accepts what needed like message, type of message ( success, error etc)